swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+from uuid import uuid4
+import math
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, age=0, id=None, condition=0):
+        self.age = age
+        self.id = uuid4().int if id is None else id
+        self.condition = condition
+        
+
+    def get_category(self):
+        return "Item"
+    
+    
+    def __str__(self):
+        return f"An object of type {self.get_category()} with id {self.id}."
+    
+    
+    def condition_description(self):
+        CONDITION_DESCRIPTIONS = {0: "awful. the worst item you can possibly imagine",
```

Comment:
For this static data, you could define it outside this function in the main scope of the module so that it doesn't get recreated every time the function is invoked.